mne/tests/test_epochs.py

```diff
@@ -5273,3 +5273,30 @@ def test_empty_error(method, epochs_empty):
         pytest.importorskip("pandas")
     with pytest.raises(RuntimeError, match="is empty."):
         getattr(epochs_empty.copy(), method[0])(**method[1])
+
+
+def test_drop_all_epochs():
+    """Test on_drop_all parameter in Epochs.drop."""
+    # Create tiny dummy data (3 epochs)
+    data = np.random.randn(3, 1, 10)
```

Comment:
use a random state to avoid touching the global seed.
```suggestion
    data = np.random.RandomState(0).randn(3, 1, 10)
```